### ISSUE TITLE
Auth with UID

### ISSUE THREAD

**Issue by zwned** [glauth/glauth#291]
Using the default [sample-simple config](https://raw.githubusercontent.com/glauth/glauth/e7195ed983715383bd5df1dc521f8c17d34d6a50/v2/sample-simple.cfg) with the docker container glauth/glauth-plugins, is it possible to bind to glauth using "serviceuser" account? something like `ldapsearch -LLL -H ldap://localhost:389 -D serviceuser,dc=glauth,dc=com -w mysecret -x -bdc=glauth,dc=com` this succeeds however, `ldapsearch -LLL -H ldap://localhost:389 -D serviceuser -w mysecret -x -bdc=glauth,dc=com` does not and returns: `ldap_bind: Invalid credentials (49)`

basically trying to use a bare UID rather than a full DN and Im failing miserably

**Comment by Fusion** [comment]
Right. I think you are referring to something that can typically be done using AD/SASL. This is not supported at this time.
Is there a compelling reason why you could not login using the service account DN?

**Comment by zwned** [comment]
Limitations/design choices of the application unfortunately. I ended up manually patching in support for what I needed.
